src/LoRa.cpp

```diff
@@ -174,7 +175,7 @@ int LoRaClass::endPacket()
 
   // clear IRQ's
   writeRegister(REG_IRQ_FLAGS, IRQ_TX_DONE_MASK);
-
+  writeRegister(REG_IRQ_FLAGS, IRQ_TX_DONE_MASK);
```

Comment:
This error was fixed with https://github.com/arduino/mkrwan1300-fw/commit/329dae43bff41616fd846b073c952e4ecf2d8cd2